docs/sources/structure/topic-types/scenario/index.md

```diff
@@ -0,0 +1,126 @@
+
+## Scenario structure
+
+Use this structure to keep scenarios consistent across products while allowing flexibility for scope and narrative.
+
+- Topic title: Name the situation and goal. Use either a verb phrase or a noun phrase. For example, "Investigate a CPU spike during a deployment" or "CPU spike investigation during deployment." Choose the form that best communicates the real-world context.
+
+- Overview: State the situation, the goal, and what the reader can do after completing the scenario. Highlight key decisions they'll make. Example: "By the end, you can identify the source of a CPU spike and decide whether to scale or optimize."
+
+- Before you begin: List prerequisites such as assumptions, access, roles, data available, and any baseline knowledge expected.
+
+- Approach (stages or decision points): Organize the body as a sequence of stages or micro-scenarios. For each stage:
+  - Context: What you observe and why it matters.  
+  - Actions: What to try first. Link to tasks for steps and to concepts for background. Keep inline steps minimal.  
+  - Decision: What to conclude or choose based on evidence.  
+  - Principle (optional): The transferable idea, such as "start with symptom → scope → isolate source."
+
+- Verify outcomes: Describe what "good" looks like and how to tell you're done. For example, "p95 latency returns under target and the alert clears."
+
+- Next steps: Link to related tutorials for learning and to tasks for implementation.
+
+## Audience and scope
+
+Scenarios are for users who have completed onboarding and want to apply what they've learned to real environments. Assume readers have a functioning setup with access, permissions, and live data. Unlike tutorials, which use canned datasets, scenarios use the reader's own data and context.
+
+Scenario topics can vary in scope. Write micro scenarios that focus on a single decision, such as "Alert fired: Is it noisy or real?", or end-to-end scenarios that cover broader workflows, such as "Respond to a latency regression after a deploy." Choose the smallest scope that produces a clear, transferable outcome.
+
+## Narrative and style
+
+Follow these guidelines for narrative and style:
+
+- Use direct second person ("you") and keep the voice instructional, not fictional. A light narrative frame is fine when it clarifies context, but avoid over-storytelling.  
+- Focus on decisions, signals, and trade-offs. State the principle behind each step to help readers apply it in new situations.  
+- Keep step lists short and link to tasks for detail. Keep explanations concise and link to concepts for depth.
+
+For navigation, place scenarios under a "Scenarios" section within each product's docs or near "Learn" content, depending on your IA. Link scenarios from related concepts, tutorials, and tasks to create a learning path.
```

Comment:
I think this is important enough to deserve its own heading. I know if I was wondering about adding scenarios to my docs, this would be one of my top questions.